contracts/HydraDelegation/IDelegation.sol

```diff
@@ -11,11 +11,39 @@ interface IDelegation is IWithdrawal {
 
     error InvalidMinDelegation();
 
+    /**
+     * @notice Changes the minimum delegation amount
+     * @dev Only callable by the admin
+     * @param newMinDelegation New minimum delegation amount
+     */
+    function changeMinDelegation(uint256 newMinDelegation) external;
+
     /**
      * @notice Returns the total delegation amount
      */
     function totalDelegation() external view returns (uint256);
 
+    /**
+     * @notice Claims rewards for delegator for staker
+     * @param staker Address of the validator
+     */
+    function claimDelegatorReward(address staker) external;
```

Comment:
Why do you think that it is necessary to re-order the methods here?

Reply:
Make it in the same order as it is in the contract and split the external and public functions so it is more clear!